tests/test_integration/test_data.py

```diff
@@ -19,99 +19,91 @@
 
 from elasticsearch import Elasticsearch
 
+user_mapping = {
+    "properties": {"name": {"type": "text", "fields": {"raw": {"type": "keyword"}}}}
+}
 
-def create_flat_git_index(client: Elasticsearch, index: str) -> None:
-    # we will use user on several places
-    user_mapping = {
-        "properties": {"name": {"type": "text", "fields": {"raw": {"type": "keyword"}}}}
-    }
-
-    client.indices.create(
-        index=index,
-        body={
-            "settings": {
-                # just one shard, no replicas for testing
-                "number_of_shards": 1,
-                "number_of_replicas": 0,
-                # custom analyzer for analyzing file paths
-                "analysis": {
-                    "analyzer": {
-                        "file_path": {
-                            "type": "custom",
-                            "tokenizer": "path_hierarchy",
-                            "filter": ["lowercase"],
-                        }
-                    }
-                },
-            },
-            "mappings": {
-                "properties": {
-                    "description": {"type": "text", "analyzer": "snowball"},
-                    "author": user_mapping,
-                    "authored_date": {"type": "date"},
-                    "committer": user_mapping,
-                    "committed_date": {"type": "date"},
-                    "parent_shas": {"type": "keyword"},
-                    "files": {
-                        "type": "text",
-                        "analyzer": "file_path",
-                        "fielddata": True,
-                    },
+FLAT_GIT_INDEX: Dict[str, Any] = {
+    "settings": {
+        # just one shard, no replicas for testing
+        "number_of_shards": 1,
+        "number_of_replicas": 0,
+        # custom analyzer for analyzing file paths
+        "analysis": {
+            "analyzer": {
+                "file_path": {
+                    "type": "custom",
+                    "tokenizer": "path_hierarchy",
+                    "filter": ["lowercase"],
                 }
+            }
+        },
+    },
+    "mappings": {
+        "properties": {
+            "description": {"type": "text", "analyzer": "snowball"},
+            "author": user_mapping,
+            "authored_date": {"type": "date"},
+            "committer": user_mapping,
+            "committed_date": {"type": "date"},
+            "parent_shas": {"type": "keyword"},
+            "files": {
+                "type": "text",
+                "analyzer": "file_path",
+                "fielddata": True,
             },
+        }
+    },
+}
+
+GIT_INDEX: Dict[str, Any] = {
+    "settings": {
+        # just one shard, no replicas for testing
+        "number_of_shards": 1,
+        "number_of_replicas": 0,
```

Comment:
So I know this was already here, but this seems pointless. After all examples are here to share good practices. And I would be surprised if this resulted in significant savings.

Reply:
Yes, I agree, I'll remove it.